internal/venice-common/src/main/java/com/linkedin/venice/writer/VeniceWriter.java

```diff
@@ -1588,7 +1599,8 @@ private String getSizeReport(int serializedKeySize, int serializedValueSize, int
     return "Key size: " + serializedKeySize + " bytes, " + "Value size: " + serializedValueSize + " bytes, "
         + "Replication Metadata size: " + replicationMetadataPayloadSize + " bytes, " + "Total payload size: "
         + (serializedKeySize + serializedValueSize + replicationMetadataPayloadSize) + " bytes, "
-        + "Max available payload size: " + maxSizeForUserPayloadPerMessageInBytes + " bytes.";
+        + "Max available payload size: " + maxSizeForUserPayloadPerMessageInBytes + " bytes"
+        + ((maxRecordSizeBytes == -1) ? "" : ", Max Venice record size: " + maxRecordSizeBytes + " bytes") + ".";
```

Comment:
Checking - Do we need one more set of parenthesis for the `else` part? When `maxRecordSizeBytes == -1`, what will this string resolve to?
```
Max available payload size: XYZ bytes.
```
Or 
```
Max available payload size: XYZ bytes, -1 bytes.
```

Reply:
I tested it and it actually works without the parenthesis, but I would add the parenthesis just to be clearer.

In any case, it's been revised to:
```java
+ ", Max record size: "
+ ((maxRecordSizeBytes == UNLIMITED_MAX_RECORD_SIZE) ? "unlimited" : maxRecordSizeBytes) + " bytes.";
```